src/dispatching/update_listeners.rs

```diff
@@ -139,25 +140,31 @@ pub fn polling_default(bot: Bot) -> impl UpdateListener<RequestError> {
 /// See also: [`polling_default`](polling_default).
 ///
 /// [`GetUpdates`]: crate::requests::GetUpdates
-pub fn polling(
-    bot: Bot,
+pub fn polling<R, E>(
```

Comment:
```suggestion
pub fn polling<R>(
```

Reply:
`E` is used below (e.g., `impl UpdateListener<E>`).